src/test/java/duke/EventTest.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class EventTest {
+
+    @Test
+    public void cannotCreateEventWithoutDateTime() {
+        assertThrows(DukeException.class, () -> {
+            Event event = Event.create("event");
+        });
+    }
+
+    @Test
+    public void cannotCreateEmptyEvent() {
```

Comment:
create_emptyEvent_throwsException follows the convention

Reply:
Thank you for the feedback.